controller/controller.go

```diff
@@ -272,13 +280,17 @@ func CreateProvisionedPVInterval(createProvisionedPVInterval time.Duration) func
 		if c.createProvisionedPVBackoff != nil {
 			return fmt.Errorf("CreateProvisionedPVBackoff cannot be used together with CreateProvisionedPVInterval")
 		}
+		if c.createProvisionerPVLimiter != nil {
+			return fmt.Errorf("CreateProvisionedPVInterval cannot be used together with CreateProvisionedPVLimiter")
+		}
 		c.createProvisionedPVInterval = createProvisionedPVInterval
 		return nil
 	}
 }
 
 // CreateProvisionedPVBackoff is the configuration of exponential backoff between retries when we create a
 // PV object for a provisioned volume. Defaults to linear backoff, 10 seconds 5 times.
+// If PV is not saved after given number of retries, corresponding storage asset (volume) is deleted!
 // Only one of CreateProvisionedPVInterval+CreateProvisionedPVRetryCount or CreateProvisionedPVBackoff
 // can be used.
```

Comment:
still want to deprecate this?
Deprecated: Use CreateProvisionedPVLimiter instead.

Reply:
Added